Formula/convox.rb

```diff
@@ -0,0 +1,15 @@
+class Convox < Formula
+  desc "The convox AWS PaaS CLI tool"
+  homepage "https://convox.com/"
+  url "https://bin.equinox.io/a/9RUvivxx6Lm/convox-20160606232025-darwin-amd64.tar.gz"
```

Comment:
It depends on how upstream have designed their code upstream. Could you point me to that so I can give you the best answer here? Thanks!